hyperspy/misc/eds/utils.py

```diff
@@ -18,6 +18,10 @@ def _get_energy_xray_line(xray_line):
         line]['energy (keV)']
 
 
+def _get_xray_lines_family(xray_line):
+    return xray_line[:xray_line.find('_') + 2]
```

Comment:
If you know what this does, can you add a docstring? It's not needed for the documentation, but still useful for us devs :)